src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -491,6 +491,12 @@
   <data name="ShortcutGuide_OverlayOpacity.Header" xml:space="preserve">
     <value>Opacity of background</value>
   </data>
+  <data name="ShortcutGuide_ExcludedApps.Text" xml:space="preserve">
+    <value>Excluded apps</value>
+  </data>
+  <data name="ShortcutGuide_ExcludedApps_TextBoxControl.Header" xml:space="preserve">
+    <value>To exclude an application add its name here (one per line).</value>

```

Comment:
```suggestion
    <value>To exclude an application add its name here (one per line). On these apps Win key doesn't open Shortcutguide.</value>
```

Or something similar.

Reply:
What about `To exclude an application add its name here (one per line). Shortcut Guide won't react to Windows key for these apps.`?